lib/internal/event_target.js

```diff
@@ -73,7 +73,7 @@ class Event {
       return name;
```

Comment:
Also, @BridgeAR shouldn't this return `this`? (as well as the other one)

Reply:
Yes, that would be ideal. `this` will just let `inspect()` to properly know what to do.